src/components/Posts/ComposePost.tsx

```diff
@@ -10,6 +10,7 @@ import EmojiPickerIconButton from "../Common/EmojiPickerIconButton";
 
 type ComposePostProps = {
   placeholder: string;
+  onClose: () => void;
```

Comment:
Can we make this an optional prop instead? Like this
```suggestion
  onClose?: () => void;
```

Reply:
Nice, so just adding a question mark before make it conditional? I have implemented the change

Reply:
Nice. And yes it makes an optional type. It's basically the same as `() => void | undefined`, so the onClose property is allowed to be undefined